### PR TITLE
Improving ssh.py

### PR DESCRIPTION
**Pull request by skateforever**
Added some new hardening checks on ssh.py and remove /etc/sshd_config to correct error when this file does not exist.

### CODE REVIEW

File: HnTool/modules/ssh.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def __init__(self, options):
         self.short_name="ssh"
         self.long_name="Checks security problems on sshd config file"
         self.type="config"
-        self.required_files = ['/etc/ssh/sshd_config', '/etc/sshd_config']
```
Comment (gomex) [inline HnTool/modules/ssh.py]: Why did you remove /etc/sshd_config ?

Comment (skateforever) [inline HnTool/modules/ssh.py]: 'Cause that config file when does not exist the ssh module not perform and when you create it duplicate the output.

Comment (skateforever) [inline HnTool/modules/ssh.py]: The correct approach on hntool is to ask the config file's full path, but I don't know how to do that yet!

Comment (hdoria) [inline HnTool/modules/ssh.py]: @gomex I think it's safe to apply this PR. Any objections?

Comment (gomex) [inline HnTool/modules/ssh.py]: Make sense to me too.
